spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/classic/ClassicAppRegistryCommands.java

```diff
@@ -84,15 +84,17 @@ public boolean availableWithCreateRole() {
 
 	@CliCommand(value = APPLICATION_INFO, help = "Get information about an application")
 	public List<Object> info(
```

Comment:
I believe we wanted to make the `--name` and `--type` an additional option to avoid breaking things....